### DISCUSSION TITLE
Can shared albums be created?

### DISCUSSION THREAD

**Discussion by AlexisMaizel** [RhetTbull/osxphotos#1403] [Q&A]
Is it possible to script the creation of iCloud Shared albums?

**Comment by RhetTbull** [comment]
No. Unfortunately I do not believe it is possible to script the creation of shared albums nor is it possible to add photos to a shared album.  You could add photos to a regular album (and even create the album) with osxphotos then manually drag those photos to the shared album.  This is a limitation of the Photos API and there's nothing osxphotos can do to work around this (as far as I know).

**Comment by AlexisMaizel** [comment]
Thanks. I had to ask :-) Fantastic tool anyhow!
